src/helpers/loadDocument.js

```diff
@@ -80,7 +80,7 @@ const extractXodOptions = options => {
   }
 
   if (options.streaming !== null) {
-    xodOptions.streaming = options.streaming === 'true';
+    xodOptions.streaming = options.streaming;
```

Comment:
it looks like if we have streaming set on constructor, the value is a string and then it gets set here.

before we had 
xodOptions.streaming = options.streaming === 'true';

so if we call loadDocument with streaming set to true,

then options.streaming === 'true' becomes false.